src/libraries/System.Text.Json/src/System.Text.Json.csproj

```diff
@@ -215,6 +216,15 @@
     <Compile Include="System\Text\Json\Writer\Utf8JsonWriter.WriteValues.String.cs" />
     <Compile Include="System\Text\Json\Writer\Utf8JsonWriter.WriteValues.UnsignedNumber.cs" />
   </ItemGroup>
+
+  <ItemGroup Condition="'$(TargetArchitecture)' != 'wasm'">
+    <Compile Include="System\Text\Json\Reader\JsonReaderHelper.SpeedOpts.cs" />
+  </ItemGroup>
+  
+  <ItemGroup Condition="'$(TargetArchitecture)' == 'wasm'">
```

Comment:
Will this work for switching for `wasm`? (i.e. does it build as a different arch; or do I need to add TFMs to the .csproj to encourage it)

Reply:
Trying `TargetsBrowser` instead